composer.json

```diff
@@ -8,7 +8,7 @@
         "php"                       : ">=7.2.5",
         "guzzlehttp/guzzle"         : "^6.5.6||^7.4.3",
         "guzzlehttp/oauth-subscriber": "~0.3",
-        "symfony/validator"         : "4.*||5.*"
+        "symfony/validator"         : ">=4.0 <=7.0"
```

Comment:
This will still not allow 7.0.x versions, you can check it here:
https://semver.madewithlove.com/?package=symfony%2Fvalidator&constraint=%3E%3D4.0.0+%3C%3D7.0.0

My suggestion is to use `symfony/validator:"4.0 - 7.0"`, you can check the result here:
https://semver.madewithlove.com/?package=symfony%2Fvalidator&constraint=4.0+-+7.0

Reply:
it should hv included 7.0.0 atleast.
anyway changed to 4-7 but  git  still  uses 5.4  for  the  php83  tests